src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs

```diff
@@ -233,14 +254,20 @@ public enum CngUIProtectionLevels
     }
     public sealed partial class ECDsaCng : System.Security.Cryptography.ECDsa
     {
-        public ECDsaCng() {}
-        public ECDsaCng(int keySize) {}
-        public ECDsaCng(CngKey key) {}
+        public ECDsaCng() { }
+        public ECDsaCng(int keySize) { }
+        public ECDsaCng(System.Security.Cryptography.CngKey key) { }
+        public ECDsaCng(System.Security.Cryptography.ECCurve curve) { }
```

Comment:
I could go either way, really.  The static Create method in Core won't be calling this type; and in Desktop the safest bet is probably default-ctor+ImportParameters.  So there's not a pressing need for it.  And we sort of don't want people to pay attention to the implementation types (unless doing something like named-key interop).  But... I sort of want it now that you pointed out it isn't there...

If we want to get it into RTM we won't need to change the contract version number for it, just add it at the same time as the implementation.